examples/plot_intelex.py

```diff
@@ -0,0 +1,273 @@
+"""
+Creating models that are accelerated by Intel(R) Extension for scikit-learn
+--------------------------------------------------------------------------------
+
+Introduction
+============
+
+This guide demonstrates how Intel(R) Extension for scikit-learn (also
+``scikit-learn-intelex``, or ``sklearnex``) can be used to speed up inference of
+Scikit-learn models.
+
+The extension supports most of Scikit-learn's classical machine learning
+algorithms, like k-nearest neighbors, support vector machines, linear/logistic
+regression, and more. Stock Scikit-learn implementations are used where no
+optimized version is available, making this package 100% compatible with
+existing code. An up-to-date list of supported algorithms can be found in the
+`official documentation
+<https://intel.github.io/scikit-learn-intelex/algorithms.html>`_.
+
+Intel(R) Extension for Scikit-learn uses the `Intel(R) oneAPI Data Analytics
```

Comment:
I basically copied a paragraph from an older Medium blog post. In the updated version it is replaced by a more high level description and links to two Medium blog posts (for anyone who needs more details). I first wasn't sure if I'm supposed to link to Medium in this example, but it might just be the best solution.